### PR TITLE
docs: fix image ref in README

### PR DESCRIPTION
**Pull request by danielpacak**
Signed-off-by: Daniel Pacak <pacak.daniel@gmail.com>

## Initial Checklist

- [ ] There is an issue describing the need for this PR.
- [x] Git log contains summary of the change.
- [x] Git log contains motivation and context of the change.
- [ ] If part of an EPIC, PR git log contains EPIC number.
- [ ] If part of an EPIC, PR was added to EPIC description.

## Description (git log)

N/A

## Type of change

- [x] Bug fix (non-breaking change fixing an issue).
- [ ] New feature (non-breaking change adding functionality).
- [ ] Breaking change (cause existing functionality not to work as expected).

## How Has This Been Tested?

N/A

## Final Checklist:

Pick "Bug Fix" or "Feature", delete the other and mark appropriate checks.

- [ ] I have made corresponding changes to the documentation.
- [ ] My code follows the style guidelines (C and Go) of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented all functions/methods created explaining what they do.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix, or feature, is effective.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published before.

## Git Log Checklist:

My commits logs have:

- [ ] Subject starts with "subsystem|file: description".
- [ ] Do not end the subject line with a period.
- [ ] Limit the subject line to 50 characters.
- [ ] Separate subject from body with a blank line.
- [ ] Use the imperative mood in the subject line.
- [ ] Wrap the body at 72 characters.
- [ ] Use the body to explain what and why instead of how.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -87,19 +85,18 @@ Hostname: ubuntu-impish
...
 ```shell
-docker run \
-  --name tracee --rm -it \
+docker run --name tracee --rm -it \
   --pid=host --cgroupns=host --privileged \
   -v /etc/os-release:/etc/os-release-host:ro \
   -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host \
-  aquasec/tracee:latest \
-  trace
+  -e TRACEE_EBPF_ONLY=1 \
```
Comment (danielpacak) [inline Readme.md:98]: I've added this temporarily, until we bring back the trace subcommand.

Comment (yanivagman) [reply Readme.md:98]: @rafaeldtinoco I also see that in your PR you removed these lines:

> ## Documentation
> 
> The full documentation of Tracee is available at [https://aquasecurity.github.io/tracee/latest](https://aquasecurity.github.io/tracee/latest). You can use the version selector on top to view documentation for a specific version of Tracee.

I think that we need to get them back

Comment (danielpacak) [reply Readme.md]: Thanks for heads up. It's hard to follow what is going on so I just fixed image ref in scope of this PR

**Review by rafaeldtinoco — APPROVED** [review]
LGTM
